oci/private/push.sh.tpl

```diff
@@ -44,4 +45,8 @@ REFS=$(mktemp)
 for tag in "${TAGS[@]+"${TAGS[@]}"}"
 do
   "${CRANE}" tag $(cat "${REFS}") "${tag}"
-done
\ No newline at end of file
+done
+
+if [[ -e "${TAGS_FILE:-}" ]]; then
+  cat "${TAGS_FILE}" | xargs -n1 "${CRANE}" tag $(cat "${REFS}")
```

Comment:
oh, we should probably use coreutils for these. could be done as a follow-up for all rules though.

Reply:
Yeah or we could read the `{{tags}}` file if it exists and just populate the `TAGS` array, then they'll be picked up in the loop above. there's not enough bats testing for this file so I'm scared to make that change right now

Reply:
I agree. we could do that as a follow-up.